cypress/support/component.ts

```diff
@@ -22,6 +22,7 @@ import './commands';
 import { mount } from 'cypress/react';
 import '@testing-library/cypress';
 import '@testing-library/cypress/add-commands';
+import 'cypress-plugin-tab';
```

Comment:
❗ 
- The last release of the package was 4 years ago
- It doesn't seem to be used in our tests
- Wasn't added to package.json

Reply:
We use it in a couple of places.

Reply:
Yep you are right.